contracts/SpokePool.sol

```diff
@@ -1167,7 +1167,7 @@ abstract contract SpokePool is
         uint32 rootBundleId,
         USSSpokePoolInterface.USSRelayerRefundLeaf calldata relayerRefundLeaf,
         bytes32[] calldata proof
-    ) public virtual override nonReentrant {
+    ) public payable virtual override nonReentrant {
```

Comment:
So this doesn;'t compile if we leave this change out?

Reply:
yea exactly